IBX-411: Users segments (#122)
ezsystems/user-documentation: docs/personalization/recommendation_models.md

    If you do not want to group recommendations based on a certain attribute, 
    remove the attribute parameter from the request. 
    The submodel is then omitted.

## Segments

> Segments allow to get personalized content suitable for particular users groups. They compute models based on segment attribute factor.

Reviewer: ```suggestion
With segments, you can get personalized content that is suitable for particular users groups. 
Segments compute models based on segment attribute factor.
```
Is it "segments" or... "models" maybe?

> Information with user segment are provided in each event which comes from the tracking script.

Reviewer: ```suggestion
Information about user segment is provided in each event that comes from the tracking script.
```

> First, make sure you have [enabled personalization](https://doc.ibexa.co/en/latest/guide/personalization/enabling_personalization/#enable-personalization) and [configured item type tracking](https://doc.ibexa.co/en/latest/guide/personalization/recommendation_client/#set-up-item-type-tracking).

Reviewer: ```suggestion
To use segments, you must first [configure item type tracking](https://doc.ibexa.co/en/latest/guide/personalization/recommendation_client/#set-up-item-type-tracking).
```
Enabling perso seems self-explanatory, we're halfway through an article about perso configuration.

If your [user role](../site_organization/organizing_the_site.md#permissions) includes 
the `Segment/All functions`, `Segment group/All functions` Policies, you can configure segment settings in the models according to your requirements.
To do this, go to the **Models** section and click the **Edit** icon next to a name of the model.

> With segments groups you can assign users to different recommendation groups based on data gathered and deliver recommendations to these user groups.

Reviewer: ```suggestion
With segment groups you can assign users to different recommendation groups based on data gathered, and deliver recommendations to these user groups.
```

> The **Segment** list displays only active segments and is generated from the events collected for relevant history (the actual data from recommendation engine, not what is added using the Back Office).

Reviewer: ```suggestion
The **Segment** list displays active segments only and is generated from the events collected for relevant history (the actual data from the recommendation engine, and not data that you upload through the Back Office).
```

Author: It's not uploaded, it's manually added.
